### DISCUSSION TITLE
Welcome to JARVIS-MARK5 Discussions!

### DISCUSSION THREAD

**Discussion by Likhithsai2580** [Likhithsai2580/JARVIS-MARK5#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TonyStark1109** [comment]
Hi, I am Gurumani from Malaysia, recently i saw your project regarding the jarvis mark 5 and i loved it i try to follow your instructions and run the program but unfortunately its not working can you help me out with it please, as i have additional features of my own to work with it

**Comment by Likhithsai2580** [comment]
yes. please share in issues

**Reply by TonyStark1109** [reply]
I have done all the step and also install all the requirements you have ask but then there is this package called psycopg2 i have already installed yet it shows the an error

**Reply by anony45-everywhere** [reply]
try creating venv

**Reply by TonyStark1109** [reply]
Yes i create it using venv and running in it , but then still the same maybe I missed out one or 2 instructions or something I'm not sure could do like a video to install this project, could be really helpful

**Reply by anony45-everywhere** [reply]
can send logs

**Reply by TonyStark1109** [reply]
Yes sure
![Screenshot_2025-01-03-01-40-59-006_com.miui.gallery.jpg](https://github.com/user-attachments/assets/8e6b699b-3b56-420d-88f7-20774f47c747)

**Reply by anony45-everywhere** [reply]
you can use psycopg2-binary library instead of psycopg2. after installation the usage is the same.

**Reply by TonyStark1109** [reply]
Actually yes i tried both of it but it's still the same, i tried each one of it

**Reply by anony45-everywhere** [reply]
can mind joining in discord server where i can see live

**Reply by anony45-everywhere** [reply]
also other who are having issue with this can also find helpful

**Reply by anony45-everywhere** [reply]
also i think you are using python 3.13 can u try degrading to python 3.12

**Reply by anony45-everywhere** [reply]
also try this if you want to use python 3.13 https://pypi.org/project/psycopg/

**Reply by anony45-everywhere** [reply]
hope it helps @TonyStark1109 gn

**Reply by TonyStark1109** [reply]
Alright sure can you send me the discord server and yes I'm using python 3.13, oh so it does need python 3.12

**Reply by Likhithsai2580** [reply]
https://discord.gg/2BGD2Cuk34

**Reply by TonyStark1109** [reply]
Thank you, is it ok if i show it around tomorrow, live with you?

**Reply by Likhithsai2580** [reply]
sry i cant, i will be going to hostel :(

**Comment by arunbalaji123** [comment]
Hey everyone,

I’ve been working with the “osanseviero/mistral-super-fast” repo, following the setup instructions, but ran into an issue when trying to integrate it into my project. It turns out that the model no longer exists on Hugging Face, even though it was referenced in the repo. When I checked manually, it wasn't available anymore, leading to this error:

""""""""""""ValueError: Hugging Face API key not found!"""""""""""""""
I tried to recover by looking for alternative models and switched to a similar one that I thought might work, but unfortunately, that didn’t resolve the issue. I’m still facing the same problem with no access to the desired model, and none of the recovery steps have helped so far.

Has anyone encountered this situation or found a good alternative for this kind of task? Any advice would be really appreciated!

Thanks! 🙏

**Reply by Likhithsai2580** [reply]
instead of jarvis mark 5, jarvis-re-j4e

**Reply by arunbalaji123** [reply]
why bro... there no way to resolve.?

**Reply by Likhithsai2580** [reply]
well hugging space is down. i have to rework on it but now i on another project xcceleratedl

**Reply by arunbalaji123** [reply]
nice bro... are u have any idea  abt this... i mean when u will rework bro..?
and if u dont mind.. i had sent u a DM in insta... shall we talk there..?

**Reply by Likhithsai2580** [reply]
Sure
